tests/Kernel/fixtures/patterns/date_block_pattern_rendering.yml

```diff
@@ -0,0 +1,50 @@
+# Test "default" variant.
+- variant: default
+  date:
+    start: 370173600
+    end: NULL
+    timezone: Europe/Brussels
+  assertions:
+    count:
+      div.ecl-date-block.ecl-date-block--default: 1
+      div.ecl-date-block.ecl-date-block--cancelled: 0
+      div.ecl-date-block.ecl-date-block--past: 0
+    equals:
+      span.ecl-date-block__week-day: Thu
+      span.ecl-date-block__day: 24
+      span.ecl-date-block__month: Sep
+      span.ecl-date-block__year: 1981
+
+# Test "cancelled" variant.
```

Comment:
We forget to check the variant "ecl-date-block ecl-date-block--ongoing"

Reply:
Fixed.